### ISSUE TITLE
Interface style Circular Sky

### ISSUE THREAD

**Issue by hisako87** [breezy-weather/breezy-weather#688]
### Describe your idea

So Geometric weather finally stopped working for me 2 days ago, and I found Breezy Weather... Wondering if there are any plans to add the circular sky layout?

The material animation (default) interface style is huge on my screen, with a lot of blank space above the actual weather data. My phone has a wide aspect ratio and this doesn't look good.

At the very least I would appreciate being able to scale down the font size, but would love the circular sky layout back... Geometric weather was my favorite.

Thank you

### Other details

Geometric weather, succinct:
![Screenshot_20240217-134542](https://github.com/breezy-weather/breezy-weather/assets/13172016/c7a72315-62a5-435e-b5f4-ffb1fcc8fbf3)

Breezy weather, very in my face:
![Screenshot_20240217-134638](https://github.com/breezy-weather/breezy-weather/assets/13172016/e0f283fd-ad09-48e1-9582-2c46ebd83726)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] My idea is not related to a question covered in the [Frequently Asked Questions](https://github.com/breezy-weather/breezy-weather/blob/main/HELP.md) document, or a feature described in the [Homepage](https://github.com/breezy-weather/breezy-weather/blob/main/docs/HOMEPAGE.md) document.
- [X] I have updated the app to **[version 4.6.14 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
Duplicate of #154 

Tl;dr: circular sky was removed years ago by the developer of Geometric Weather. There are options to turn off animations and remove current details in Settings > Home screen if you don't like them.
